blocks/into/into.css

```diff
@@ -0,0 +1,26 @@
+.into{
+    max-width: 2560px;
```

Comment:
Done✅

Reply:
@NazarPauk Can you please delete the `max-width: 1200px`; from your `intro` block? Since we've already applied `max-width: 1200px;` to our `.page__container`, there's no need to duplicate the restriction for sections. =)